lib/internal/errors.js

```diff
@@ -1434,8 +1434,7 @@ E('ERR_UNSUPPORTED_DIR_IMPORT', "Directory import '%s' is not supported " +
 E('ERR_UNSUPPORTED_ESM_URL_SCHEME', (url) => {
   let msg = 'Only file and data URLs are supported by the default ESM loader';
   if (isWindows && url.protocol.length === 2) {
-    msg += '. Absolute Windows paths without prefix are not valid URLs, ' +
-      "consider using 'file://' prefix";
+    msg += ". Absolute Windows paths need a 'file://' prefix.";
```

Comment:
Not an objection, little clarification: I worded it this way to at least try to tell the user a reason "why" this error is present (and how to find out more) instead of just telling how to solve it.

Reply:
In that case maybe this still accomplishes that?:

> Absolute Windows paths are not valid URLs without a file:// prefix.

Reply:
Yeah, I think that would be better :+1:

Reply:
OK, done.